src/EditorFeatures/CSharpTest/UsePatternMatching/CSharpAsAndNullCheckTests_FixAllTests.cs

```diff
@@ -58,6 +58,57 @@ int M()
 
         return o is string e ? 1 : 0;
     }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineTypeCheck)]
+        public async Task FixAllInDocument2()
+        {
+            await TestInRegularAndScriptAsync(
+@"class Symbol
+{
+    public ContainingSymbol { get; }
+
+    void M(object o, bool b0, bool b1)
+    {
+        {|FixAllInDocument:var|} symbol = o as Symbol;
+        if (symbol != null)
+        {
+            while ((object)symbol != null && b1)
+            {
+                symbol = symbol.ContainingSymbol as Symbol;
+            }
+
+            if ((object)symbol == null || b2)
+            {
+                throw null;
+            }
+
+            var use = symbol;
+        }
+    }
+}",
+@"class Symbol
+{
+    public ContainingSymbol { get; }
+
+    void M(object o, bool b0, bool b1)
+    {
+    if (o is Symbol symbol)
+    {
```

Comment:
I'm not sure what's going on here with formatting. when I remove the "move trivia to the next statement" part, this works as expected.